content/questions/foundations/question-108.md

```diff
@@ -0,0 +1,16 @@
+---
+archetype: "questions"
+title: "Question 108"
+question: "What is one of the main benefits for using a Personal Access Token (PAT) instead of a standard username and password in GitHub authentication?"
```

Comment:
```suggestion
question: "What is one of the main benefits for using a Personal Access Token (PAT) instead of a standard username and password for GitHub authentication?"
```